.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,12 @@
+<!--
+Commit message:
+
+- Please use Feature / Add / Change / Fix for user-facing changes.
```

Comment:
Please also add an example for "add-on-develoepr-facing changes":
* `Feature: [NewGRF/Script/AI/GS]` for new modding abilities, which were not moddable before.
* `Add: [NewGRF/Script/AI/GS]` for additions to existing mod support.
* `Change: [NewGRF/Script/AI/GS]` for changing behavior of mod support.

I use "Script" for things, which apply to both "AI" and "GS". Not sure anyone else does.